mysql-test/suite/innodb/t/percona_session_temp_tablespaces_encrypt.test

```diff
@@ -104,9 +104,13 @@ DROP TABLE t6;
 connect (con3, localhost, root,,);
 connection con3;
 
---replace_column 1 ID
---replace_regex /\\#innodb_temp\\temp/\/#innodb_temp\/temp/
-SELECT * FROM INFORMATION_SCHEMA.INNODB_SESSION_TEMP_TABLESPACES ORDER BY SPACE;
+--let $assert_text = Active session temp tablespaces count should be 2
```

Comment:
The asserts should sync with disconnect con1/con2 by wait_until_count_sessions.inc with a proper parameter

Reply:
Fixed.